com.unity.render-pipelines.universal/Runtime/Unity.RenderPipelines.Universal.Runtime.asmdef

```diff
@@ -18,6 +19,11 @@
             "expression": "0.0.1",
             "define": "VISUAL_EFFECT_GRAPH_0_0_1_OR_NEWER"
         },
+        {
+            "name": "com.unity.adaptiveperformance",
+            "expression": "2.1.0-preview.1",
+            "define": "ADAPTIVE_PERFORMANCE_2_1_0_OR_NEWER"
```

Comment:
This should be 2.0.0-preview.x (7 imho) (as we will land the feature as 2.0.0 feature)
Which would put the define as ADAPTIVE_PERFORMANCE_2_0_0_OR_NEWER